bin/ci-stack.ts

```diff
@@ -7,12 +7,28 @@
  */
 
 import 'source-map-support/register';
-import { App } from '@aws-cdk/core';
+import { App, RemovalPolicy } from '@aws-cdk/core';
 import { CIStack } from '../lib/ci-stack';
 import { CIConfigStack } from '../lib/ci-config-stack';
+import { DeployAWSAssets } from '../lib/DeployAWSAssets';
 
 const app = new App();
 
-new CIConfigStack(app, 'CI-Config-Dev', {});
+const env: string = 'Dev';
```

Comment:
Why are we hard coding the env?

Reply:
This is hardcoded only for `opensearch-ci` here. This file does not exist in the other deployment

Reply:
Let think about this project as an individual entity than linking to another deployment.

Reply:
Considering this as an individual entity, this repository would be used for dev and therefore, dev

Reply:
This repository is not dev let's be clear on that. :) We should assume each repository/code is production grade and contribute accordingly.

Reply:
Changed the variable name to `defaultEnv`